hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/security/acl/IAccessAuthorizer.java

```diff
@@ -56,11 +56,20 @@ boolean checkAccess(IOzoneObj ozoneObject, RequestContext context)
     ALL,
     NONE;
     private static int length = ACLType.values().length;
+    private static ACLType[] vals = ACLType.values();
 
     public static int getNoOfAcls() {
       return length;
     }
 
+    public static ACLType getAclTypeFromOrdinal(int ordinal) {
+      if (ordinal > length - 1) {
```

Comment:
We should perhaps add a not less than zero check also , so we can throw exception correctly.